src/codeu/chat/common/User.java

```diff
@@ -104,4 +123,98 @@ private static int hash(User user) {
     hash+=user.creation.hashCode();
     return hash;
   }
+  
+  // userStatusUpdate()
+  // 
+  // Returns all the user updates in an dictionary 
+  // where keys are the user's user interests (UUIDs) 
+  // and a value is an arraylist of arraylists<String> 
+  // where the 1st arraylist keeps track of each userinterest's 
+  // newly created conversations (CONVOS_CREATED_ARRAY) and the second arraylist keeps 
+  // track of the conversations the userinterest has contributed messages to (CONVOS_CONTRIBUTED_TO_ARRAY)
+  //
+  public HashMap<Uuid, ArrayList<ArrayList<String>>> userStatusUpdate(Iterable<ConversationContext> conversations, Time updateTime){ 
```

Comment:
Just wanted to check, is there a specific reason why updateTime is type Time and not type long?